week-1/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,14 @@
-var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+function createPercentages(ments, studs, totalnum = 23) {
+  var permen = (ments / totalnum) * 100;
+  var perstu = (studs / totalnum) * 100;
+  permen = permen.toFixed(0)
+  perstu = perstu.toFixed(0)
+  return [permen, perstu]
```

Comment:
It's best practice not to return an array of various types. Can you think of a way to make this function both more re-usable and more DRY (don't repeat yourself)?

Reply:
Hey Jonny! I have pushed a revision of this exercise. It is one I ended up taking a particularly long time over so any feedback would be very much appreciated.